test/integration/serving/serving_test.go

```diff
@@ -340,6 +355,277 @@ func testControllerManager(t *testing.T, tester controllerManagerTester, kubecon
 	}
 }
 
+func testSchedulerServing(t *testing.T) {
```

Comment:
Can't we rename testControllerManager to testComponent and integrate the scheduler test there as well?